profile.proto

```diff
@@ -21,6 +21,7 @@ service Profile {
     rpc UploadProfileImage(stream UploadImageRequest) returns (UploadImageResponse) {}
     rpc getProfileByPhoneOrEmail(GetProfileByPhoneOrEmailRequest) returns (UserProfileProto) {}
 	rpc isUserAdmin(IdRequest) returns (IsUserAdminResponse) {}
+    rpc getAllProfiles(GetAllProfilesRequest) returns (GetAllProfilesResponse) {}
```

Comment:
Name your function in ways which define their purpose. rather than using getAllProfiles use getMultiipleProfile/bulkGetProfile.